index.html

```diff
@@ -926,10 +914,50 @@ <h2>The <code>application/vc+jwt</code> Media Type</h2>
   </table>
 </section>
 
-<section id="conformance"></section>
+<section id="vc-ld-jwt-media-type">
+  <h2><code>application/vc+ld+jwt</code></h2>
+  <p>
+    This specification registers the <code>application/vc+ld+jwt</code> Media Type specifically for
```

Comment:
This media type feels wrong... I don't understand why it's not `application/vc+jwt` -- is the reason because "It's JSON-LD"?

Reply:
Yes, the payload (claimset) is known to be JSON-LD, instead of "just normal JSON, with registered / private claim names".